.travis.yml

```diff
@@ -1,62 +1,58 @@
 language: php
 
 php:
-  - 5.3
-  - 5.4
-  - 5.5
-  - 5.6
-  - 7.0
-  - 7.1
-  - nightly
-  - hhvm-3.12
+    - 5.6
+    - 7.0
+    - 7.1
+    - nightly
+    - hhvm-3.12
 
 matrix:
-  allow_failures:
```

Comment:
Yes, but I find indentation with 4 spaces is better because readability is clearer. By they way, it's just a matter of preference, and while all our files are indented 4 spaces I though it was better to do the same for YAML.